nh_clinical/data/change_ward_manager_to_shift_coordinator.xml

```diff
@@ -19,72 +19,5 @@
-                <search string="User Management">
-                    <field name="name"/>
-                    <field name="login"/>
-                    <group expand="1" string="Filters">
-                        <filter name="filter_inactive" string="Inactive Users" domain="[['active','=',False]]"
-                                groups="nh_clinical.group_nhc_admin,nh_clinical.group_nhc_dev"/>
-                        <filter name="filter_hca" string="HCAs" domain="[['category_id.name','in',['HCA']]]"/>
-                        <filter name="filter_nurse" string="Nurses" domain="[['category_id.name','in',['Nurse']]]"/>
-                        <filter name="filter_shift_coordinator" string="Shift Coordinators" domain="[['category_id.name','in',['Shift Coordinator']]]"/>
-                        <filter name="filter_senior_manager" string="Senior Managers" domain="[['category_id.name','in',['Senior Manager']]]"
-                                groups="nh_clinical.group_nhc_admin,nh_clinical.group_nhc_dev"/>
-                        <filter name="filter_doctor" string="Doctors" domain="[['category_id.name','in',['Doctor']]]"
-                                groups="nh_clinical.group_nhc_admin,nh_clinical.group_nhc_dev"/>
-                        <filter name="filter_kiosk" string="Kiosk" domain="[['category_id.name','in',['Kiosk']]]"
-                                groups="nh_clinical.group_nhc_admin,nh_clinical.group_nhc_dev"/>
-                        <filter name="filter_admin" string="Admins" domain="[['category_id.name','in',['System Administrator']]]"
-                                groups="nh_clinical.group_nhc_admin,nh_clinical.group_nhc_dev"/>
-                    </group>
-                </search>
-            </field>
-        </record>
-        <record id="action_nhc_user_management_wm" model="ir.actions.act_window">
-            <field name="name">Account Administration</field>
-            <field name="res_model">nh.clinical.user.management</field>
-            <field name="view_mode">tree,form</field>
-            <field name="domain">[['category_ids.name','in',['HCA','Nurse','Shift Coordinator']]]</field>
-            <field name="search_view_id" ref="view_nhc_user_management_search"/>
-            <field name="context">{'partner_category_display': 'short'}</field>
-        </record>
-        <record id="action_nhc_user_management_admin" model="ir.actions.act_window">
-            <field name="name">Account Administration</field>
-            <field name="res_model">nh.clinical.user.management</field>
-            <field name="view_mode">tree,form</field>
-            <field name="domain">[['login','not in',['adt','admin']]]</field>
-            <field name="search_view_id" ref="view_nhc_user_management_search"/>
-            <field name="context">{'partner_category_display': 'short'}</field>
-        </record>
-        <!-- Overrides of records in wizard/responsibility_allocation_wizard.xml -->
```

Comment:
At the time I thought this was no longer necessary because current clients have a fresh database but you rightly pointed out that if old clients came back this could be a problem.

I do think we could work around that and still get to clean up the codebase though. Lets discuss it :)

Reply:
Ah okay now that I've looked into it I understand the real reason I did not keep this override. I looked in the original file and saw that the override made no difference because the original record already had 'Shift Coordinator' in it. This could be a mistake from the original changes.

Reply:
This is also sorted :)